src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -164,7 +165,10 @@ export default class OrganizationDiscover extends React.Component {
           });
         }
 
-        this.setState({data, isFetchingQuery: false});
+        this.setState({
```

Comment:
What's been going on with our prettier?

Reply:
In that case prettier wasn't the culprit, I actually formatted that line